src/Symfony/Bundle/FrameworkBundle/Tests/CacheWarmer/RouterCacheWarmerTest.php

```diff
@@ -12,19 +12,19 @@
 namespace Symfony\Bundle\FrameworkBundle\Tests\CacheWarmer;
 
 use PHPUnit\Framework\TestCase;
-use Psr\Container\ContainerInterface;
 use Symfony\Bundle\FrameworkBundle\CacheWarmer\RouterCacheWarmer;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\CacheWarmer\WarmableInterface;
 use Symfony\Component\Routing\RouterInterface;
 
 class RouterCacheWarmerTest extends TestCase
 {
     public function testWarmUpWithWarmableInterfaceWithBuildDir()
     {
-        $containerMock = $this->getMockBuilder(ContainerInterface::class)->onlyMethods(['get', 'has'])->getMock();
+        $containerMock = new ContainerBuilder();
```

Comment:
The variable name is not accurate anymore. Moreover, do we need a builder here?

```suggestion
        $container = new Container();
```